unit-testing-react-what-why-how/Button/problem/src/.codesandbox/template.json

```diff
@@ -1,10 +1,9 @@
 {
-  "title": "Exercise 1",
-  "description": "First exercise in the UX of Data Fetching Training",
+  "title": "Button",
+  "description": "Part of the “Unit Testing React: What, Why, How” training.",
   "iconUrl": "https://www.bitovi.com/favicon.ico",
   "tags": [
-    "Typescript",
-    "React"
+      "html"
```

Comment:
This folder is in the wrong place. It should be in the `problem` folder. I can tell because when I open it in CodeSandbox, I don’t see the info in this file.